pkg/daemon/coreos.go

```diff
@@ -50,3 +73,52 @@ func getRootBlockDeviceSysfs() (string, error) {
 	}
 	return "", fmt.Errorf("Failed to find %s", root)
 }
+
+func logAlephInformation() error {
+	f, err := os.Open(alephPath)
+	if err != nil {
+		// We assume this one should exist on CoreOS systems; if it somehow doesn't we'll just
+		// log the error but continue anyways.
+		return err
+	}
+	stat, err := f.Stat()
+	if err != nil {
+		return err
+	}
+	contents, err := ioutil.ReadAll(f)
+	if err != nil {
+		return err
+	}
+	var alephData coreosAleph
+	if err := json.Unmarshal(contents, &alephData); err != nil {
+		return err
+	}
+	fmt.Printf("CoreOS aleph version: mtime=%v build=%v imgid=%v\n", stat.ModTime().UTC(), alephData.Build, alephData.Imgid)
+	return nil
+}
+
+func logInitionProvisioning() error {
+	contents, err := ioutil.ReadFile(ignitionProvisioningPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			fmt.Printf("No %s found", ignitionProvisioningPath)
+			return nil
+		}
+		return err
+	}
+	var ignProvisioning ignitionReport
+	if err := json.Unmarshal(contents, &ignProvisioning); err != nil {
+		return err
+	}
+	fmt.Printf("Ignition provisioning: time=%v\n", ignProvisioning.ProvisioningDate)
```

Comment:
is there are reason to not use glog here?

Reply:
Makes sense, done!